lib/protect.ex

```diff
@@ -56,71 +61,95 @@ defmodule Protect do
       raise "--rules must be a json file"
     end
 
-    Enum.each([options[:org], options[:user]], &(
-      if &1 && Regex.match?(~r/[^a-zA-Z0-9\-]/, &1) do
+    Enum.each(
+      [options[:org], options[:user]],
+      &if &1 && Regex.match?(~r/[^a-zA-Z0-9\-]/, &1) do
         raise "user/org must be a valid Github username"
       end
-    ))
+    )
 
     options
   end
 
   @doc """
     Gets a list of Github repos for an owner that can be either a user or an
-    organisation.
+    organization.
   """
   def get_repos(options, page \\ 1, repos \\ []) do
     new_repos =
       options
       |> get_repos_url(page)
-      |> Github.get!
+      |> Github.get!()
       |> Map.get(:body, "{}")
-      |> Poison.decode!
+      |> Poison.decode!()
       |> Enum.map(&Map.fetch!(&1, "name"))
 
     case length(new_repos) < 100 do
       true -> repos ++ new_repos
-      _ -> get_repos options, page + 1, repos ++ new_repos
+      _ -> get_repos(options, page + 1, repos ++ new_repos)
+    end
+  end
+
+  defp owner(options) do
+    cond do
+      options[:org] ->
+        options[:org]
+
+      options[:user] ->
+        options[:user]
     end
   end
```

Comment:
I think we can simplify this function to:
```suggestion
  defp owner(options), do: options[:org] || options[:user]
```